src/controllers/AbstractRestController.ts

```diff
@@ -0,0 +1,29 @@
+import { inject } from 'inversify';
+import { BaseHttpController, httpGet, controller } from 'inversify-express-utils';
+import { RedisService } from '../services/RedisService';
+import { TYPES } from '../InjectableTypes';
+
+@controller('/')
+export abstract class AbstractRestController extends BaseHttpController {
+
+    constructor(@inject(TYPES.RedisService) protected readonly redisService: RedisService) {
+        super();
+    }
+
+    @httpGet('healthCheck')
+    public async healthCheck(): Promise<void> {
+        await this.sendInfo();
+    }
+
+    private async sendInfo(): Promise<void> {
+        const obj: any = await this.redisService.checkHealth();
+        this.httpContext.response.send(
+            `
+            <p>Redis online:  ${this.redisService.ping()}</p>
+            <p>Number of Times Visited: ${obj.n}\n</p>
```

Comment:
Health check should communicate correctness through status code so observers does not have to parse response. It usually does 200 for OK and 500 for not OK.